1st-gen/scripts/confirm-build.js

```diff
@@ -42,38 +54,47 @@ async function verifyCustomElementsJson() {
     return Promise.all(checks);
 }
 
-function verifyVersionJs() {
+function verifyVersionTs() {
```

Comment:
The `version.ts` is now living under 2nd-gen, therefore this verification script got updated.